contents/handbook/people/team-structure/platform.md

```diff
@@ -13,62 +13,48 @@ hideAnchor: true
 
 ## Mission
 
-Make using and developing for PostHog as reliable as running water. Wherever you want it.
-
-## Goals 
-
-- We don't lose events
-- Data is as up to date as possible
-- Engineers always be able to ship and build
-- Fail fast. Fix faster.
-- Ship anywhere
-- Stack scales with demand
-- Support Small Teams (and contributors) in building and debugging PostHog
-- Be frugal.
-
-## Responsibilities
-Concrete things we take responsibility over:
-
-- [app.posthog.com](https://app.posthog.com) and its infrastructure
-- On Prem & Single Tenant deployments
-- CI/CD - How we deploy
-- Data infrastructure (ClickHouse, Kafka)
-- Monitoring and Alerting stack
-- Event ingestion
-- Extensibility: Plugin server, plugins, and libraries
-
-## Customer
-
-- Other Small Teams in making sure they have the tools (databases, queues, etc) and the ability to deploy effortlessly that they need to build
-- End users (Both cloud and on-prem teams)
-
-## Output metrics
-
-### VPC
-###### Retention 
-- Metric: Retention
-- Objective: Better than cloud
-### Cloud
-###### Data Loss
-- Metric: Data loss %
-- Objective: < 0.1%
-###### Uptime 
-- Metric: Uptime
-- Objective: > 99.99%
-###### Speed 
-- Metric: Speed
-- Objectives
-  - Event ingestion: TBD
-  - Query response: TBD
-- Overall: We should anticipate increasing demand (either manually or automatically)
-##### Cost 
-- Metric: Infra Costs 
-- Objective: Our costs should grow at a rate that is sublinear relative to scale
-### Dev experience
-##### Dev experience NPS (Infra)
-- Metric: Developer experience (relating to infra) (maybe NPS?)
-- Objective: TBD (maybe NPS?)
+Make deploying, scaling and managing PostHog easy
+
+## Goals & Key Results
+
+* PostHog is the easiest self-hosted product to deploy and scale in the world
+    * Result: Volume of clients requring help @marcushyett-ph
```

Comment:
I like having the accountability of having peoples names here. It's also very inline with us being transparent

Reply:
This is just for the metrics - you can remove your name when you've got a link to it - but I do agree it would be nice to have an ongoing owner for each.